include/unix/osd.h

```diff
@@ -61,5 +62,18 @@ static inline int ofi_close_socket(int socket)
 	return close(socket);
 }
 
+static inline int ofi_sockerr()
```

Comment:
These functions should all have `(void)` as their argument lists instead of `()`.